tests/dashboard/test_order.py

```diff
@@ -1220,6 +1220,16 @@ def test_render_cancel_fulfillment_page(admin_client, fulfilled_order):
     assert response.status_code == 200
 
 
+def test_view_edit_order_customer_note(admin_client, order_with_lines):
+    url = reverse("dashboard:order-edit-customer-note", kwargs={"order_pk": order_with_lines.pk})
+    note_content = "this is a note"
+    data = {"csrfmiddlewaretoken": "hello", "customer_note": note_content}
+    response = admin_client.post(url, data)
+    assert response.status_code == 200
+    order_with_lines.refresh_from_db()
+    assert order_with_lines.customer_note == note_content
```

Comment:
Can you do a `get()` of the order from the dashboard and assert the status code is 200? The rendering crashes.

Reply:
you mean do `admin_client.get(...)` on the `Order detail API` or the `Edit Customer's note API` ?

Reply:
On the order details page of the order we edited.

Reply:
We need to retrieve the order details page to ensure it is rendering